src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -790,24 +789,18 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
 
             **Examples:**
 
-            `appName RLIKE 'z.*|q.*''`
+            `appName RLIKE r'z.*|q.*''`
 
             * `z-app`
             * `q-app`
-
-            `hostname RLIKE 'ip-10-351-[0-2]?[0-9]-.*'`
-
+            *`hostname RLIKE r'ip-10-351-[0-2]?[0-9]-.*'`
```

Comment:
I don't think we want a `*` here for the bullet point, this should be above the bulleted section

Reply:
Got it, will make the change!